specs/unit/validation/place-order-request-validator.specs.ts

```diff
@@ -219,11 +260,20 @@ function createValidDeliveryAddress(): Address {
     };
 }
 
-function createValidOrderItems(): Array<BasketItem & { price: number; }> {
+function createValidOrderItems(): Array<BasketItem & { price: number; tags: string[] }> {
     return [{
         name: 'Some pizza',
         price: 10,
         quantity: 1,
-        version: 'Large'
+        version: 'Large',
+        tags: ['pizza']
     }];
+}
+
+function createSide(name: string, quantity: number, price: number): BasketItem & { price: number } {
```

Comment:
I wouldn't have added this function because it actually makes the code less readable. Just put the data inline for now and we'll refactor later.